### PR TITLE
TST: Don't modify master security lists directory during tests

### PR DESCRIPTION
**Pull request by jikamens**
Rather than drop files temporarily into the master security lists
directory during unit tests, create temporary directories for the
tests. This avoids issues when the tests are being run at the same
time as other code that uses the real security lists data.

### CODE REVIEW

**Comment by jikamens** [comment]
@ehebert @twiecki this is to fix a problem with parallel qexec builds. It's a pretty straightforward change. Can you take a look?

File: zipline/utils/test_utils.py [hunk 1]
```diff
@@ -115,15 +117,34 @@ def nullctx():
...
+    old_dir = security_list.SECURITY_LISTS_DIR
+    new_dir = tempfile.mkdtemp()
+    try:
+        for subdir in os.listdir(old_dir):
+            shutil.copytree(os.path.join(old_dir, subdir),
+                            os.path.join(new_dir, subdir))
+            with patch.object(security_list, 'SECURITY_LISTS_DIR', new_dir), \
+                    patch.object(security_list, 'using_copy', True,
+                                 create=True):
+                yield
```
Comment (twiecki) [inline zipline/utils/test_utils.py]: Haven't seen this before, does it not have to `yield` something?

Comment (jikamens) [inline zipline/utils/test_utils.py]: When you decorate a method with `@contextmanager`, everything up until the yield becomes the eqiuvalent of the `__entering__` function for the context manager and everything after the yield becomes the `__exit__` function.

Comment (jikamens) [inline zipline/utils/test_utils.py]: https://docs.python.org/2/library/contextlib.html

Comment (twiecki) [inline zipline/utils/test_utils.py]: That's cool, thanks for explaining.

**Comment by twiecki** [comment]
Looks good to me :+1:

**Pull request merged**